crates/ruma-client-api/src/r0/profile/set_avatar_url.rs

```diff
@@ -75,7 +88,7 @@ mod tests {
                     .uri("https://bar.org/_matrix/client/r0/profile/@foo:bar.org/avatar_url")
                     .body(&[] as &[u8]).unwrap(),
             ).unwrap(),
-            IncomingRequest { user_id, avatar_url: None } if user_id == "@foo:bar.org"
+            IncomingRequest { user_id, avatar_url: None, blurhash: None} if user_id == "@foo:bar.org"
```

Comment:
You can use `, ..` instead of `, blurhash: None` here to make the test compile both with and without `unstable-pre-spec`.